### ISSUE TITLE
Capability to reload the config

### ISSUE THREAD

**Issue by Ren60FHk** [hSaria/ChromaTerm#71]
There was the **-r** option in the version 0.3.4.
```
./ct -h   
ChromaTerm-- v0.3.4
Usage: ./ct [-adr] [-c file]
     -a       Allow colliding actions
     -c file  Override configuration file
     -d       Demo the available custom color-codes
     -r       Reload the config file for all CT instances
```
It was very helpful and it would be nice if you could implement such a mechanism for recent version (0.4.x).

**Comment by hSaria** [comment]
Fair enough. I didn't know people used this, but I'll add it back.

**Comment by Ren60FHk** [comment]
> Fair enough. I didn't know people used this, but I'll add it back.

Thanks again! )

**Comment by hSaria** [comment]
My pleasure. Keep the suggestions coming :)

**Comment by Ren60FHk** [comment]
> My pleasure. Keep the suggestions coming :)

I will :)

**Comment by hSaria** [comment]
Completed in eeb3f8d59fd83ad36444be1825570412ad8eb28d. Will be published in v0.4.2.
